src/components/formControl/index.tsx

```diff
@@ -23,22 +28,35 @@ const FormControl: FC<PropsWithChildren<Props>> = ({
   id,
   label,
   hint,
+  tooltip,
 }) => {
   const isInvalid = !!errorMessage;
 
+  const renderLabel = (
```

Comment:
The `render` prefix feels a bit off for your variables as they do not render but only hold the JSX partial. In a react context I woul associate the `render` keyword with a function that renders the view.